packages/cadl-ranch-specs/http/payload/multipart/mockapi.ts

```diff
@@ -0,0 +1,19 @@
+import { passOnSuccess, ScenarioMockApi, mockapi, ValidationError } from "@azure-tools/cadl-ranch-api";
+import { jpgFile } from "../../helper.js";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+Scenarios.Payload_MultiPart_FormData_basic = passOnSuccess(
+  mockapi.post("/multipart/form-data/mixed-parts", (req) => {
+    req.expect.deepEqual(req.body.id, "123");
+    if (req.files?.length > 0) {
+      req.expect.deepEqual("profileImage", req.files[0].fieldname);
+      req.expect.deepEqual("application/octet-stream", req.files[0].mimetype);
+      req.expect.deepEqual(jpgFile, req.files[0].buffer);
```

Comment:
left is "actual", right is "expected"

there seems a bug on
https://github.com/Azure/cadl-ranch/blob/8f28f9227b86fb21ef2a120f1c0ee1fbf965229f/packages/cadl-ranch-api/src/expectation.ts#L92
`ValidationError` got left as "expected", right as "actual"

Reply:
fixed